.github/workflows/create-release.yml

```diff
@@ -103,3 +80,68 @@ jobs:
             docker images
 
             make push GITHUB_ACTIONS=${GITHUB_ACTIONS} GITHUB_REPOSITORY_OWNER=${{ github.repository_owner }}
+
+  bundle:
+    runs-on: ubuntu-latest
+    container: "ghcr.io/wtsi-npg/ub-16.04-irods-clients-4.2.7:latest"
+
+    needs: deploy
+
+    if: always()
+
+    steps:
+    - name: install exodus
+      run: |
+        apt-get update
+        apt-get install -q -y python3-pip
+        pip3 install exodus-bundler
+
+    - name: bundle clients
+
+      run: exodus --tarball --add /usr/lib/irods/plugins/ --add /lib/x86_64-linux-gnu/ --output client_bundle.tgz /usr/local/bin/baton-* /usr/bin/ils /usr/bin/iinit /usr/bin/imeta
+
+    - name: upload bundle
+      uses: actions/upload-artifact@v3
+      with:
+        name: client_bundle
+        path: "client_bundle.tgz"
+
+  release:
+    runs-on: ubuntu-latest
+
+    needs: bundle
+
+    if: always()
```

Comment:
Is this because of an Actions idiosyncracy? It looks like a conditional that's always true, but I guess you needed to add it?

Reply:
Yes, I found that when I didn't put the `if` here (as well as on the bundling), it did the bundling, then skipped the release if building the containers had been skipped, for some reason...